docs/development/quickstart.md

```diff
@@ -0,0 +1,65 @@
+# Development quickstart
```

Comment:
Can you include:
```
---
layout: default
nav_order: 1
parent: Development
---
```
And probably update the order of the other pages?

Reply:
Ah, good catch, I totally forgot about front-matters here.